tests/python/gaia-ui-tests/gaiatest/tests/functional/clock/test_clock_set_alarm.py

```diff
@@ -14,8 +14,10 @@ def setUp(self):
         self.clock.launch()
 
     def test_clock_set_alarm(self):
+        # Added a view switching method to app.py for getting a Clock object.
```

Comment:
```
# Added a view switching method to app.py for getting a Clock object.
```
I'm not sure this comment has a lot of value. This looks like duplication with the Page Class. If somebody needs to know why the test changed that way. `git log tests/python/gaia-ui-tests/gaiatest/tests/functional/clock/test_clock_set_alarm.py` can be used, and it'll point to that bug and this PR.